src/components/FormHelp.js

```diff
@@ -0,0 +1,34 @@
+import _ from 'underscore';
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+
+const propTypes = {
+    /** Content of FormHelp */
+    // eslint-disable-next-line react/forbid-prop-types
+    children: PropTypes.any.isRequired,
+};
+
+const FormHelp = (props) => {
+    if (_.isEmpty(props.children)) {
+        return null;
+    }
+
+    return (
+        <View
+            style={[
+                styles.mt1,
+                styles.flexRow,
+                styles.justifyContentBetween,
+                styles.baseTextInputPaddingHorizontal,
+            ]}
+        >
+            {props.children}
+        </View>
+    );
+};
+
+FormHelp.propTypes = propTypes;
+FormHelp.displayName = 'FormHelp';
+export default FormHelp;
```

Comment:
`FormHelp` feels like over-engineering to me. I think that using a ternary in `BaseTextInput` to render either `errorText` or `hint` would be much simpler and less confusing.

Reply:
`FormHelp` is just the empty container to apply the default padding and margin and if there is no child then just noop it.

Reply:
If you think this is not needed, I can move the View to TextInput itself.

Reply:
Please let me know what do you think?

Reply:
Yea, I think we should move it.